### PR TITLE
Iteration 6

### PR DESCRIPTION
**Pull request by c-brenn**
🔵

### CODE REVIEW

File: panacea/lib/panacea/pml/ast.ex [hunk 1]
```diff
@@ -9,6 +9,28 @@ defmodule Panacea.Pml.Ast do
     do_unquote(ast, 0)
   end
 
+  def decode(ast) do
```
Comment (c-brenn) [inline panacea/lib/panacea/pml/ast.ex]: should probably add an `encode` as well and use that in `Analysis`

File: panacea/test/controllers/asclepius_controller_test.exs [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+      action foo {
+        requires { drug { "torasemide" } }
+      }
+      action bar {
+        requires { drug { "trandolapril" } }
+      }
+    }
+    """
+    {:ok, ast} = Panacea.Pml.Parser.parse(pml)
+    ast
```
Comment (c-brenn) [inline panacea/test/controllers/asclepius_controller_test.exs]: use `Ast.encode` here

File: panacea/web/controllers/ast_controller.ex [hunk 3]
```diff
@@ -2,28 +2,11 @@ defmodule Panacea.AstController do
   use Panacea.Web, :controller
 
   def to_pml(conn, %{"ast" => encoded_ast}) do encoded_ast
```
Comment (c-brenn) [inline panacea/web/controllers/ast_controller.ex]: formatting here lol

**Comment by c-brenn** [comment]
fixed the above in #170 .

Looks good to my 👀

**Review by houli — APPROVED** [review]

**Pull request merged**